packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -279,6 +279,44 @@ endpoint_tests:
       - master
       - agent
 
+######### /system/checks/ endpoint
+  - tests:
+      are_upstream_req_headers_ok:
+        skip_authcookie_filtering_test: true
+        auth_token_is_forwarded: skip
+        test_paths:
+          - /system/checks/v1/node/
+          - /system/checks/v1/cluster/
+    type:
+      - agent
+  - tests:
+      are_upstream_req_headers_ok:
+        skip_authcookie_filtering_test: false
+        auth_token_is_forwarded: skip
+        test_paths:
+          - /system/checks/v1/node/
+          - /system/checks/v1/cluster/
+    type:
+      - master
+  - tests:
+      is_response_correct:
+        expect_http_status: 200
+        nocaching_headers_are_sent: skip
```

Comment:
Let's set it to false and be explicit. The `skip` option was meant as a stop-gap for legacy endpoints. For newly added endpoints we can make the tests actually check if no-caching headers are not set.